vision/module/module_orientation.py

```diff
@@ -0,0 +1,80 @@
+"""
+module_orientation will calculate the orientation of the module in degrees
+and return them as a tuple (x tilt, y tilt) using a derivative through the x- and y- axes
+"""
+import numpy as np
+import argparse
+from region_of_interest import region_of_interest
```

Comment:
Lets move these two imports into the if __name__ == '__main__' so they are not confused as necessary for the whole program.